contracts/interfaces/IPool.sol

```diff
@@ -20,6 +20,8 @@ struct Asset {
 
 interface IPool {
 
+  function swapOperator() external view returns (address);
```

Comment:
leftover?

Reply:
I'm using this in `SafeTracker` to check who is calling the function

Reply:
Would this require a redeploy? Or is that view fn present in the pool but only missing in the interface?